README.md

```diff
@@ -58,6 +58,7 @@ Resize to `200x200px` using `embed` method and change format to `webp`:
 | position / pos | [Docs](https://sharp.pixelplumbing.com/api-resize#resize)       | `/s_200x200,pos_top/buffalo.png`                     | Sets `position` option for `resize`.                                                                                                                              |
 | trim           | [Docs](https://sharp.pixelplumbing.com/api-resize#trim)         | `/trim_100/buffalo.png`                              |
 | extend         | [Docs](https://sharp.pixelplumbing.com/api-resize#extend)       | `/extend_{top}_{right}_{bottom}_{left}/buffalo.png`  | Extend / pad / extrude one or more edges of the image with either the provided background colour or pixels derived from the image.                                |
+| extend.background | \_                                                           | `/b_%23f0f0f0/buffalo.png`                           |
```

Comment:
Shouldn't it be?

```suggestion
| background | \_                                                           | `/b_%23f0f0f0/buffalo.png`                           |
```

Reply:
Seems like sharp has both `options.background` and [`extend.background`](https://sharp.pixelplumbing.com/api-resize#extend) which for the latter I thought of being the only way to set background. Your change seems fine since It seems they are doing the exact same thing